src/components/logical-expression-builder/pipes/display-value.pipe.ts

```diff
@@ -0,0 +1,62 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { LogicalExpressionBuilderService } from '../services/logical-expression-builder.service';
+import { DisplayValueFunction } from '../interfaces/DisplayValueFunction';
+import { DatePipe } from '@angular/common';
+
+@Pipe({ name: 'displayValue' })
+export class DisplayValuePipe implements PipeTransform {
+    private readonly _transformFunction: DisplayValueFunction;
+
+    constructor(private _lebService: LogicalExpressionBuilderService, private _datePipe: DatePipe) {
+        this._transformFunction = this._lebService.getDisplayValueFunction();
+    }
+
+    transform(value: any, fieldType?: string, fieldData?: any): string {
+        if (this._transformFunction) {
+            return this._transformFunction(value, fieldType, fieldData);
```

Comment:
You could allow to return undefined from the transform function, so that the standard display value is used instead